.github/workflows/build.yaml

```
@@ -13,6 +13,27 @@ permissions:
   contents: read
 
 jobs:
+  clang-format-check:
+    name: Clang Format Check
+    runs-on: macos-15
+    timeout-minutes: 10
+    steps:
+      - uses: actions/checkout@eef61447b9ff4aafe5dcd4e0bbf5d482be7e7871 # v4.2.1
+        with:
+          fetch-depth: 1
+
+      - name: Install Clang Format
+        run: brew install clang-format
+
+      - name: Run Clang Format Check
+        run: |
+          git clang-format --diff --quiet
+          if ! git ...; then
+            echo "ERROR: Code is not properly formatted."
+            echo >&2"Please run clang-format locally and commit the changes."
```

Needs a space before the string:

```suggestion
            echo >&2 "Please run clang-format locally and commit the changes."
```